README.md

```diff
@@ -1,4 +1,4 @@
-_forked off from [Ronzil's 2048 AI repo](https://github.com/ronzil/2048-AI)._
```

Comment:
Could you replace this credit? Could be at top or bottom of README, up to you

Reply:
should be fixed